client-runtime/src/main/java/com/microsoft/rest/http/HttpHeaders.java

```diff
@@ -107,6 +109,18 @@ private HttpHeader getHeader(String headerName) {
         return headers.get(headerKey);
     }
 
+    /**
+     * Convert this HttpHeaders collection to JSON.
+     * @return The JSON object that contains this HttpHeaders collection's values.
+     */
+    public JSONObject toJSON() {
```

Comment:
Any specific reasoning behind using org.json instead of `implements JsonSerializable` or `@JsonProperty` from Jackson?

Reply:
Done.